### PR TITLE
dump the needed foreign dependencies for common distros

### PR DESCRIPTION
**Pull request by Xe**
Based on comments and feedback from IRC

### CODE REVIEW

File: src/choosenim.nim [hunk 1]
```diff
@@ -40,6 +40,9 @@ proc chooseVersion(version: string, params: CliParams) =
               Warning, HighPriority)
       display("Hint:", "Install clang or gcc using your favourite package manager.",
               Warning, HighPriority)
+      display("Hint:", "The following commands might help:", Warning, LowPriority)
+
+      echoForeignDeps()
```
Comment (PMunch) [inline src/choosenim.nim]: Won't this echo no matter the verbosity level, while the above command will only show the explanation for Warning and LowPriority?

Comment (dom96) [reply src/choosenim.nim]: Just don't use the `foreignDep`, problem solved, no?

**Comment by genotrance** [comment]
Any update on this PR?

**Comment by dom96** [comment]
Please reopen if you get a chance to correct the verbosity problem.

**Pull request closed without merging**